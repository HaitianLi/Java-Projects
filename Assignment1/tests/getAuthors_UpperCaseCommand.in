add collection books.csv
LIST AUTHORS
exit