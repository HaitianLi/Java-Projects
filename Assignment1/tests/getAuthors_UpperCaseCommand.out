user: 17 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: Ending Library process.
